### PR TITLE
Add headers to SMTP client to prevent being tagged as spam

### PR DESCRIPTION
**Pull request by ishank011**

### CODE REVIEW

File: pkg/smtpclient/smtpclient.go [hunk 1]
```diff
@@ -67,14 +84,22 @@ func (creds *SMTPCredentials) sendMailAuthSMTP(recipient, subject, body string)
...
-func (creds *SMTPCredentials) sendMailSMTP(recipient, subject, body string) error {
+func (creds *SMTPCredentials) sendMailSMTP(recipient, subject, message string) error {
 
 	c, err := smtp.Dial(fmt.Sprintf("%s:%d", creds.SMTPServer, creds.SMTPPort))
 	if err != nil {
 		return err
 	}
 	defer c.Close()
 
+	host, err := os.Hostname()
```
Comment (mirekys) [inline pkg/smtpclient/smtpclient.go:95]: `os.Hostname()` may come to problems when running inside k8s containers, we would be probably better off with a configurable sender hostname (like it is done e.g. in [gomail](https://github.com/go-gomail/gomail/blob/master/smtp.go#L33-L35))

Comment (mirekys) [reply pkg/smtpclient/smtpclient.go:95]: Or maybe as a fallback, we could use domain part of the `SenderMail` as hostname?

Comment (ishank011) [reply pkg/smtpclient/smtpclient.go:95]: @mirekys makes a lot of sense. I'll make the change

File: pkg/smtpclient/smtpclient.go [hunk 2]
```diff
@@ -67,14 +103,22 @@ func (creds *SMTPCredentials) sendMailAuthSMTP(recipient, subject, body string)
...
+	if err != nil {
+		return err
+	}
+	if err = c.Hello(host); err != nil {
```
[the remark this answers is not in the document]
Comment (mirekys) [reply pkg/smtpclient/smtpclient.go:118]: looks good otherwise, thanks! 👍

[the remark this answers is not in the document]
Comment (ishank011) [reply pkg/smtpclient/smtpclient.go]: @mirekys Done, thanks!

**Pull request merged**